src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -328,15 +328,16 @@ private IList<HtmlModel> BuildHtmlModels(string basePath, IList<TocModel> tocMod
 
         private void ConvertCore(string basePath, string pdfFileName, IList<HtmlModel> htmlModels)
         {
-            var converter = new HtmlToPdfConverter(
-                htmlModels,
-                new HtmlToPdfOptions
-                {
-                    BasePath = basePath,
-                    UserStyleSheet = _pdfOptions.CssFilePath,
-                    LoadErrorHandling = _pdfOptions.LoadErrorHandling,
-                    AdditionalArguments = _pdfOptions.AdditionalPdfCommandArgs,
-                    OutlineOption = _pdfOptions.OutlineOption
+			var converter = new HtmlToPdfConverter(
```

Comment:
Replace <kbd>tab</kbd> with 4 <kbd>space</kbd>s

Reply:
Please find tabs were replaced with spaces in the commit af6b07fe862fd572a7ee76b5607bd3e6bf273816